patches/minecraft/net/minecraft/village/Village.java.patch

```diff
@@ -135,3 +170,36 @@
          }
      }
  
+@@ -553,4 +574,32 @@
+             this.field_75590_b = p_i1674_3_;
+         }
+     }
++
++    /* ======================================== FORGE START =====================================*/
++    private net.minecraftforge.common.capabilities.CapabilityDispatcher capabilities;
++    public boolean hasCapability(net.minecraftforge.common.capabilities.Capability<?> capability, @Nullable net.minecraft.util.EnumFacing facing)
++    {
++        if (getCapability(capability, facing) != null)
```

Comment:
`hasCapability` is meant to be a faster cheaper check than `getCapability` so it should not call `getCapability`.